challenges/classes.js

```diff
@@ -1,8 +1,34 @@
 // Copy and paste your prototype in here and refactor into class syntax.
 
+
+class CuboidMaker { 
+  constructor(measurements) { 
+    this.length = measurements.length,
+    this.width = measurements.width,
+    this.height = measurements.height;
+  };
+}
+
+
+CuboidMaker.prototype.volume = function() {
+  return this.length * this.width * this.height;
+}
+
+CuboidMaker.prototype.surfaceArea = function() {
```

Comment:
class syntax would involve placing these methods inside the class definition, instead of using the prototype